### DISCUSSION TITLE
How to configare https on be

### DISCUSSION THREAD

**Discussion by bingwill** [apache/doris#27950] [A - General / Q&A]
Can anybody tells me how to configure https on BE ，not only brpc but also http webserver（on port 8040）。
tks！

**Comment by LemonLiTree** [comment]
Configure port? be.conf?

**Reply by bingwill** [reply]
I want to enable https,and successfully configured on fe and it works well. Lately,I found that be also supports https,but it seems to be the brpc working in https mode which bond on port 8060 after configured the enable_https and ssl_certificate_path and ssl_private_key_path in be.conf. And the be's webserver still work in http mode on port 8040. And in conclusion，all i want is to enable be's webserver on https mode which was bond on port 8040 by default. 
tks.

**Reply by jgz-dev-gh** [reply]
Is there any update on this issue? Without HTTPS, to be honest, it pretty much eliminates any possibility to deploy in production environments in big corporations.  Any chance to prioritize this?
